README.md

```diff
@@ -30,11 +32,11 @@ This extension contains privileged code as WebExtension APIs and therefore can o
 5. Set the value to the desired branch name (see below)
 
 ##### Branch names
-- captive-portal
-- privacy-hostname
+- captive-portal: in Browser Console type `browser.experiments.vpnRecommender.fakeCaptivePortal()`
+- privacy-hostname: triggered when you visit websites, examples include disconnect.me and strongvpn.com
 - control
 - catch-all
-- streaming-hostname
+- streaming-hostname: triggered when you visit websites, examples include hulu.com and netflix.com
```

Comment:
Ah, thanks for this!

Note to future self: https://github.com/raymak/vpn-recommendation-shield-study/blob/5850eb7e1247ddb39d474b43bf3ccb9a2516c4ea/src/privileged/vpnRecommender/VpnRelatedHostnames.jsm#L80-L97